### ISSUE TITLE
Enhancement - provide feedback directly from the cli

### ISSUE THREAD

**Issue by clarkbreyman-yammer** [Azure/azure-devops-cli-extension#208]
The cli should allow feedback directly rather than forwarding to the browser for authenticated users. 

eg.
```
vsts feedback 'Submit feedback directly from the command line'.
```

**Comment by gauravsaralMs** [comment]
I will suggest we can **explore** on how can we extend `az feedback`
right now 'az feedback` can open an issue from cli (in cli main repo or cli extension repo) depending on the command for which you are giving feedback.

May be we can see if a command from devops extension is selected then instead of opening issue in cli-extension repo it can open issue in devops repo.

cc: @geverghe @snankani

**Comment by gauravsaralMs** [comment]
Opened up a request in azure cli for this:
https://github.com/Azure/azure-cli/issues/9483

**Comment by gauravsaralMs** [comment]
FYI: @atbagga @ishitam8

**Comment by geverghe** [comment]
@atbagga - You have made a fix for this in az cli right? Could you reference the fix here?

**Comment by atbagga** [comment]
Fix in PR - https://github.com/Azure/azure-cli/pull/9679

@geverghe  Should we deprecate az devops feedback in favour of az feedback once the fix goes in?

**Comment by atbagga** [comment]
PR is merged and the functionality should be available in the next Azure CLI release (2.0.68).

Closing the issue.
